hack/testdata/nats/check.sh

```
@@ -0,0 +1 @@
+return 0
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)